src/main/java/org/fundacionjala/sfdc/pageobjects/opportunities/SFODetailsPage.java

```diff
@@ -41,4 +41,9 @@ public void clickDeleteAlert() {
     public String getNewAccountSavedName() {
         return CommonActions.getElement(this.newAccountLabel).getText();
     }
+
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)